.github/ISSUE_TEMPLATE/03--issues-with-content-on-articles-and-docs.md

```diff
@@ -1,13 +1,13 @@
 ---
-name: 'Bug Report: Issues with /news or guide articles or documentation'
-about: Report issue with a specific article, like broken links, typos, missing parts, etc
+name: 'Report issues with content on /news, guide articles or documentation'
+about: Report issues with content on a specific article, like broken links, typos, missing parts, etc.
 title: ''
-labels: 'status: needs help for triage, type: bug'
+labels: 'type: bug, status: waiting triage'
```

Comment:
Do we want to update the label name then?

Reply:
Yes - You are correct. I mentioned in the OP, I intend to adjust the labels after this lands.